libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -878,7 +878,7 @@ bool AC_Fence::sys_status_enabled() const
     if (!sys_status_present()) {
         return false;
     }
-    if (_action == AC_FENCE_ACTION_REPORT_ONLY) {
+    if (_action == uint8_t(AC_FENCE_ACTION::REPORT_ONLY)) {
```

Comment:
```suggestion
    if (_action == static_cast<uint8_t>(AC_FENCE_ACTION::REPORT_ONLY)) {
```

Reply:
@Ryanf55 san, @peterbarker san.
I changed the cast to static_cast<AC_Fence::Action>.

Reply:
Excellent.